wrapper/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+#------------------------------------------------------------------------------
```

Comment:
This file should have a license header.

Reply:
Just remembered that the license header is only used in source code.